sycl/include/sycl/detail/pi.h

```diff
@@ -1835,6 +1836,16 @@ __SYCL_EXPORT pi_result piTearDown(void *PluginParameter);
 /// runtime must handle it or end the application.
 __SYCL_EXPORT pi_result piPluginGetLastError(char **message);
 
+/// Returns the global timestamp from \param device , and syncronized host
```

Comment:
what does it mean that host timestamp is synchronized?
```suggestion
/// Returns the global timestamp from \param device , and synchronized host
```

Reply:
As in the host timestamp that corresponds with the device timestamp being returned

Reply:
Do you mean that both timestamps are noted at the same time? This might not be true for backends that would have different calls to obtain such data.

Reply:
They may not be necessarily noted at the same time, but as close as possible. I'm going off the definition of `HostTime` given in OpenCL :

>host_timestamp will be updated with the value of the host timer in nanoseconds at the closest possible point in time to that at which device_timer was returned.

Reply:
The OpenCL is much more clear to me, please rephrase your comment.